packages/matter.js/src/behavior/definitions/door-lock/DoorLockServer.ts

```diff
@@ -4,11 +4,19 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/*** THIS FILE WILL BE REGENERATED IF YOU DO NOT REMOVE THIS MESSAGE ***/
-
+import { DoorLock } from "../../../cluster/definitions/index.js";
```

Comment:
Please import the specific file and not "all clusters" via index.

```suggestion
import { DoorLock } from "../../../cluster/definitions/DoorLockCluster.js";
```